packages/mariadb/package.json

```diff
@@ -39,15 +39,9 @@
     "@sequelize/core": "workspace:*",
     "@sequelize/utils": "workspace:*",
     "dayjs": "^1.11.10",
-    "lodash": "4.17.21",
-    "mariadb": "3.3.0",
-    "semver": "7.6.0",
-    "wkx": "0.5.0"
-  },
-  "devDependencies": {
```

Comment:
needed by https://github.com/sequelize/sequelize/pull/17252

Reply:
If it's okay with you I'd prefer to add them back in that PR

Reply:
Seeing as that PR is also good to merge, I'll just update this afterwards. Saves you a merge conflict there